### PR TITLE
Remove PoiCard and display full poi details

### PR DESCRIPTION
**Pull request by G-Ray**
## Description
Remove the `PoiCard` component and its associated `css` code. Display full poi's details instead. This behavior will change as soon as we introduce a re-sizable panel for a poi, which will be done in another PR.

The "header" of panel has been refactored. This might introduce visual glitches I did not noticed yet.

## Screenshots
Peek 2020-07-29 15-47

### CODE REVIEW

File: src/panel/poi/PoiPanel.jsx [hunk 1]
```diff
@@ -268,28 +252,22 @@ export default class PoiPanel extends React.Component {
...
-          <span className="poi_panel__back_text">{backAction.text}</span>
-        </div>
-      }
-      {isFromPagesJaunes(poi) && <img className="poi_panel__pj_logo"
-        src="./statics/images/pagesjaunes.svg"
-        alt="PagesJaunes" />
-      }
-    </div>;
+    const header = backAction &&
+      <div className={backAction.className} onClick={backAction.callback}>
```
Comment (bbecquet) [inline src/panel/poi/PoiPanel.jsx:258]: Maybe it's possible to use the `<Flex>` component here to get rid of even more CSS.

Comment (G-Ray) [reply src/panel/poi/PoiPanel.jsx]: yep, it's done, thanks

File: src/components/ui/Flex.jsx [hunk 2]
```diff
@@ -1,19 +1,29 @@
-import React from 'react';
+import React, { forwardRef } from 'react';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 
-const Flex = ({ children, inline, className,
+const Flex = forwardRef(({
```
Comment (G-Ray) [inline src/components/ui/Flex.jsx:5]: I used forwardRef and added rest props

File: src/panel/poi/PoiPanel.jsx [hunk 3]
```diff
@@ -157,20 +155,6 @@ export default class PoiPanel extends React.Component {
     window.app.navigateTo('/favs');
   }
 
-  showDetails = () => {
-    const poi = this.getBestPoi();
-    Telemetry.add(Telemetry.POI_SEE_MORE, null, null,
```
Comment (bbecquet) [inline src/panel/poi/PoiPanel.jsx:162]: The corresponding constant (`poi_see_more`) in `local_modules/telemetry/index.js` can be removed.

File: src/panel/poi/PoiPanel.jsx [hunk 4]
```diff
@@ -268,28 +253,21 @@ export default class PoiPanel extends React.Component {
         text: _('Back to list'),
         className: 'poi_panel__back_to_list',
```
Comment (bbecquet) [inline src/panel/poi/PoiPanel.jsx:254]: Nit: There is only one version left of this, so this could be put inline in the `Flex` element.

**Pull request merged**